RealmSwift/Util.swift

```diff
@@ -47,6 +47,110 @@ internal func gsub(pattern: String, template: String, string: String, error: NSE
                                            withTemplate: template)
 }
 
+// MARK: ObjectiveCBridgeable
+
+// Used for conversion from Objective-C types to Swift types
+internal protocol ObjectiveCBridgeable  {
+    /* FIXME: Remove protocol once SR-2393 bridges all integer types to `NSNumber`
```

Comment:
That wasn't the impression I got talking with Joe today. Let me follow up before we change or merge this.